roles/tableau/tasks/config.yml

```diff
@@ -95,10 +95,6 @@
   shell: "netstat -tnp | grep httpd | awk '{print $4}' | tail -1 | awk -F: '{print $2}'"
   register: server_port
 
-- name: "TABLEAU CONFIG | Configure Tableau Server gateway port on node0 to {{ tableau_port }}"
```

Comment:
I removed this task because the default node in current version of Tableau server is `node1`. I'm guessing `node0` was holdover from a previous version.